igvc_gazebo/nodes/ground_truth/main.cpp

```diff
@@ -0,0 +1,108 @@
+#include <ros/ros.h>
+#include <nav_msgs/Odometry.h>
+#include <mutex>
+#include <geometry_msgs/Vector3.h>
+#include <tf/transform_broadcaster.h>
+#include <tf/transform_listener.h>
+#include <tf/transform_datatypes.h>
+#include <igvc_utils/NodeUtils.hpp>
+#include <robot_localization/navsat_conversions.h>
+
+ros::Publisher ground_truth_pub;
+std::mutex mutex;
+nav_msgs::Odometry og_pose;
+ros::Time last_estimate;
+
+void odom_callback(const nav_msgs::Odometry::ConstPtr& msg) {
```

Comment:
In general, function and class method names are camelCased.

http://wiki.ros.org/CppStyleGuide#Function_.2BAC8_Methods